src/desktop/isns.desktop

```diff
@@ -13,6 +13,7 @@ X-SuSE-YaST-AutoInst=all
 X-SuSE-YaST-Geometry=
 X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=isns
+X-SuSE-YaST-Keywords=internet,storage,name,network,server,devices,iscsi
```

Comment:
I'm not sure about `name`, that's maybe too generic. On the other hand I'd also add plain `scsi` as it is related to iSCSI.